source/reference/about-mongosync.txt

```diff
@@ -0,0 +1,131 @@
+About ``mongosync``
+========================
+
+The ``mongosync`` binary is the primary process used in {+c2c-product-name+}. 
+``mongosync`` migrates data from one cluster to another and keeps the 
+clusters in continuous sync. You can use ``mongosync`` to create dedicated 
+analytics, development, or testing clusters that mirror your production 
+environment. 
+
+``mongosync`` enters different states depending on the requests it receives. 
+The current ``mongosync`` state determines which API operations you can run. 
+For more information on ``mongosync`` states, see 
+:ref:`mongosync States <c2c-states>`.
+
+.. note:: 
+
+   Most state transitions happen as the result of an API call. The exceptions 
+   are:
+
+   - The transition from ``COMMITTING`` to ``COMMITTED``
+   - The transition from ``REVERSING`` to ``RUNNING``
+
+How ``mongosync`` Works
+------------------------
+
+Initial Connection
+~~~~~~~~~~~~~~~~~~~
+
+``mongosync`` must create an initial connection to the source and destination 
+clusters before it can start to sync data. For more information, see 
+:ref:`c2c-connecting`.
+
+When ``mongosync`` connects to the source and destination clusters, it is in the 
+:ref:`IDLE <c2c-state-idle>` state.  
+
+During Sync 
+~~~~~~~~~~~~
+
+The :ref:`start <c2c-api-start>` endpoint begins the 
+:ref:`RUNNING <c2c-state-running>` state. 
+
+If you start a new sync, ``mongosync`` syncs data to the destination cluster. 
+If you make subsequent writes to the source cluster, ``mongosync`` applies them 
+to the destination cluster.  
+
+For every collection on the source, ``mongosync`` creates a corresponding 
+collection on the destination. 
+
+``mongosync`` temporarily alters some collection characteristics during 
+the ``RUNNING`` state and changes these back to their source 
+cluster characteristics during the ``COMMIT`` state. For more information, see
+:ref:`Commit Alters <c2c-commit-alters>`.
+
+Collection Copy
+'''''''''''''''
+
+Collection copy is the initial full data transfer that copies each collection 
+from the source cluster to the destination cluster. ``mongosync`` performs 
+collection copy by splitting each source collection into partitions, gathering 
+all partitions from all collections into one list, and copying all partitions 
+to their corresponding destination collections.
+
+Change Event Application
+'''''''''''''''''''''''''
+
+Change event application (CEA) is an ongoing process that continuously updates 
+the destination cluster when there are changes to the source cluster. If there 
+are multiple ``mongosync`` instances, all the ``mongosync`` instances finish 
+collection copy before entering CEA.
+
+During CEA, ``mongosync`` receives :ref:`change stream <changeStreams>` events 
+from the source cluster, converts these events into writes, and applies these 
+writes on the destination cluster. CEA ensures both :ref:`CRUD <crud>` and 
+:ref:`DDL <c2c-mongosync-ddl>` operations are handled accurately.
```

Comment:
[suggestion] for active voice

```suggestion
:ref:`DDL <c2c-mongosync-ddl>` operations execute accurately.
```

Reply:
Rephrasing the question for clarity, does "accurately" mean that if a write operation is applied on the source, the operation executes as expected on the source or does it mean that if a write operation is applied on the source, it is synced as intended on the destination

Reply:
"Accurately" means "if a write operation is applied on the source, it is synced as intended on the destination." I just changed the wording to clarify this!